### DISCUSSION TITLE
Ensuring Universal Accessibility to GitHub Profile

### DISCUSSION THREAD

**Discussion by sarikarai29** [community/community#110998] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Question for GitHub Community 

Title: Ensuring Universal Accessibility to GitHub Profile

Body: How can I ensure universal accessibility to my GitHub profile, given that I can access it from my mobile and laptop, but encounter a 404 error from other devices? 

My GitHub link: [pushpakrai1607]
(https://github.com/pushpakrai1607) I'm facing difficulty resolving this error. Any guidance or suggestions would be greatly appreciated. Thank you.

 ![Screenshot_2024-03-05-18-00-33-13_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/community/community/assets/76905002/a025fe4b-d527-4b22-8b01-00e128dc46da)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @sarikarai29 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
